src/modules/handle-key.js

```diff
@@ -48,6 +48,11 @@ var handleKeyDown = function(event, params, modal) {
     }
   } else {
     if (keyCode === 13) {
+      if (e.ctrlKey || e.metaKey) {
+        fireClick($okButton);
```

Comment:
So this will now allow you to close confirmation `swal`s right? This will presumably be true irrespective of whether its showing an error or a success.